doc/run/example-of-machine.md

```diff
@@ -0,0 +1,118 @@
+
+### Perform model_devi task at a local Slurm cluster
+
+In this example, we perform the model_devi task at a local Slurm workstation.
+
+```json
+"model_devi": [
+    {
+      "command": "lmp",
+      "machine": {
+       "context_type": "local",
+        "batch_type": "Slurm",
+        "local_root": "./",
+        "remote_root": "/home/user1234/work_path"
+      },
+      "resources": {
+        "number_node": 1,
+        "cpu_per_node": 4,
+        "gpu_per_node": 1,
+        "queue_name": "QueueGPU",
+        "custom_flags" : ["#SBATCH --mem=32G"],
+        "group_size": 10,
+        "source_list": ["/home/user1234/lammps.env"]
+      }
+    }
+],
+```
+
+The "command" for the model_devi task in the LAMMPS is "lmp".
+
+In the machine parameter, we specify the type of job scheduling system by changing the "batch_type" to "Slurm".
+
+In the resources parameter, we specify the name of the queue to which the task is submitted by adding "queue_name". We can add additional lines to the calculation script via the "custom_flags". In the model_devi steps, there are frequently many short tasks, so we usually pack multiple tasks (e.g. 10) into a group for submission. Other parameters are similar to that of the local workstation.
+
+### Perform fp task in a remote PBS cluster
+
+In this example, we perform the fp task at a remote PBS cluster that can be accessed via SSH.
+
+```json
+"fp": [
+    {
+      "command": "mpirun -n 32 vasp_std",
+      "machine": {
+       "context_type": "SSHContext",
+        "batch_type": "PBS",
+        "local_root": "./",
+        "remote_root": "/home/user1234/work_path",
+        "remote_profile": {
+          "hostname": "39.xxx.xx.xx",
+          "username": "user1234"
+         }
+      },
+      "resources": {
+        "number_node": 1,
+        "cpu_per_node": 32,
+        "gpu_per_node": 0,
+        "queue_name": "QueueCPU",
+        "group_size": 5,
+        "source_list": ["/home/user1234/vasp.env"]
+      }
+    }
+],
+```
+
+VASP code is used for fp task and mpi is used for parallel computing, so "mpirun -n 32" is added to specify the number of parallel threads.
+
+In the machine parameter, "context_type" is modified to "SSHContext" and "batch_type" is modified to "PBS". It is worth noting that "remote_root" should be set to an accessible path on the remote PBS cluster. "remote_profile" is added to specify the information used to connect the remote cluster, including hostname, username, password, port, etc. 
```

Comment:
```suggestion
In the machine parameter, "context_type" is modified to "SSHContext" and "batch_type" is modified to "PBS". It is worth noting that "remote_root" should be set to an accessible path on the remote PBS cluster. "remote_profile" is added to specify the information used to connect the remote cluster, including hostname, username, port, etc. 
```
Password should be discouraged. Instead, we should encourage SSH keys in the doc.